### ISSUE TITLE
FEATURE - Create a companion Apple Watch app

### ISSUE THREAD

**Issue by SuzGupta** [mikaelacaron/brain-marks#132]
**Is your feature request related to a problem? Please describe.**
I am always frustrated when an app I enjoy is not available on the Watch. And by "always" I mean for the last two days, since I just started using a Watch 😆

**Describe the solution you'd like**
I'd like us to work together to build a companion app for the Watch.

**Describe alternatives you've considered**
Living without this, was going to say just using Twitter on the Watch, but I'm not sure Twitter is on the Watch?

**Additional context**
Book to help get started: https://www.raywenderlich.com/books/watchos-with-swiftui-by-tutorials/

Fellow contributors, please give this issue a thumbs-up if you too would like to have an Apple Watch version of this app!

**Comment by Jakayus** [comment]
Hi @mikaelacaron, I would like to try this out! May I have this issue?

**Comment by mikaelacaron** [comment]
@Jakayus Go for it! I think a watch app showing the saved tweets would be good, not adding any additional ones. Also I am in the process of removing AWS Amplify DataStore from the project, and instead saving all the tweets to a JSON file. I'm not sure if that will affect how you implement the watch app

**Comment by Jakayus** [comment]
@mikaelacaron Thank you for that clarification on the functionality - I also lean towards just viewing the tweets. 
Unclear whether it will affect implementation or not yet. I'm also just learning about HomeBrew and SwiftLint thanks to your repo, so I'm getting up to speed setting up environment, but those sound so cool!!

Can I ask any critical design questions that come up here, or is there a preferred method for those?

**Comment by mikaelacaron** [comment]
Feel free to ask here, or DM me on Insta or Twitter, see [links here](https://github.com/mikaelacaron)

I'll probably respond faster on Insta or Twitter haha

And yeah definitely only view tweets (and categories) on the watch, cause I don't even think it's possible, haha or would take a lot of work, to paste links into a watch, to even add them to the list

**Comment by SuzGupta** [comment]
Yeah, I was just thinking more about this too and I think the Watch app lists saved categories (with icons? without? optionally just with icons? [what about when the same icon is used for more than one category though]) and then, when you tap a category, see the s saved in that category. Tapping the start of a tweet should show the whole Tweet in some decent-looking text view. Unless there's some way one can view the Tweet in [Chirp](https://www.chirpapp.io/)?

**Comment by SuzGupta** [comment]
The Chirp screens at least provide a decent approach of how to display Tweets on the Watch.

**Comment by Jakayus** [comment]
Thank you for the ideas @SuzGupta  I checked out the tweets in the Chirp app and it is a good example! Icons based list sounds good too.

Unfortunately I did not get that far - Learned this week that AWS Amplify does not support watchOS (it is even stated on their GitHub page) and spent some time trying to formulate workarounds, but they are not very swifty approaches, and definitely not using the power of SwiftUI re-use across platforms. 

After talking with @mikaelacaron , it seems like the best approach for now is to actually wait for BrainMarks to be updated to JSON implementation (vs AWS Amplify Datastore). Then the watch can rely on the JSON it gets from the phone and make the watch design more straightforward.

**Comment by mikaelacaron** [comment]
@Jakayus thanks so much! I'll add a hold label to this one for now
